ts/features/walletV3/payment/components/WalletPaymentFeedbackBanner.tsx

```diff
@@ -0,0 +1,39 @@
+import { Banner, VSpacer } from "@pagopa/io-app-design-system";
+import { openAuthenticationSession } from "@pagopa/io-react-native-login-utils";
+import { default as React } from "react";
+import { View } from "react-native";
+import I18n from "../../../../i18n";
+import { mixpanelTrack } from "../../../../mixpanel";
+
+const WalletPaymentFeebackBanner = () => {
+  const bannerViewRef = React.useRef<View>(null);
+
+  const handleBannerPress = () => {
+    void mixpanelTrack("VOC_USER_EXIT", {
+      screen_name: "PAYMENT_OUTCOMECODE_MESSAGE"
+    });
+
+    return openAuthenticationSession(
+      "https://io.italia.it/diccilatua/ces-pagamento",
```

Comment:
What do you think about adding this URL into a constant file?

Reply:
I had already moved this link to a const, IDK what happened, maybe a merge gone wrong. I'll fix it asap!

Reply:
Fixed in https://github.com/pagopa/io-app/pull/5390/commits/336a54bf871c389fcea39cc1b0be327fb48a3137